libraries/chain/abi_serializer.cpp

```diff
@@ -401,6 +439,14 @@ namespace eosio { namespace chain {
             auto h1 = ctx.push_to_path( impl::variant_path_item{ .variant_itr = v_itr, .variant_ordinal = static_cast<uint32_t>(select) } );
             return vector<fc::variant>{v_itr->second.types[select], _binary_to_variant(v_itr->second.types[select], stream, ctx)};
          }
+
+         if(kv_tables.size() > 0 && is_string_valid_name(rtype)) {
```

Comment:
Nitpick:  I think `!kv_tables.empty()` is the idiomatic approach instead of `kv_tables.size() > 0`.  It used to be that `size()` wasn't guaranteed to be a constant operation.  I don't know if there are other compelling reasons now other than possibly it being more concise.